src/_posts/languages/python/django/2000-01-01-start.md

```diff
@@ -1,123 +1,112 @@
+
+{% note %}
+  You can also use our web dashboard to achieve this operation
+{% endnote %}
+
+```bash
+$ scalingo create myapp
+```
+
+* Go on the [dashboard](https://dashboard.scalingo.com/apps) of your application.
+* Select the __Addons__ category
+* Choose the database you want to use PostgreSQL®, MySQL®...
+
+## Initialize Your Application locally
+
+On your computer, start a Django website.
 
 ```bash
 # Create your project
 mkdir my-project
 cd my-project
 
-# Create a virtualenv
-virtualenv venv
-source venv/bin/activate
+# use pipenv to handle dependancies (recommanded for python buildpack)
+pip install --user pipenv
 
 # Install Django and other useful packages:
-pip install django django-environ psycopg gunicorn dj-database-url whitenoise
+pipenv install django django-environ psycopg gunicorn dj-static
+
+# enter the virtual environment
+pipenv shell
 
 # Initialize Django project, this command will
 # create a manage.py file and a myapp directory
-django-admin.py startproject myapp .
+django-admin startproject myapp .
 ```
 
+{% note %}
+  If you've selected MySQL database instead of PostreSQL,
+  do not install psycopg but mysqlclient instead.
+
+  `pipenv install django mysqlclient gunicorn ...`
+{% endnote %}
+
+{% note %}
+  Pipenv is the prefered way to manage dependencies. It will had Pipfile and Pipfile.lock
+  files, therefor deployment system will consider your application as a Python app.
+
+  [Please refere to Python buildpack for more information on this
+  subject]({% post_url languages/python/2000-01-01-start %})
+{% endnote %}
+
 ## Define How To Start Your Application
 
-By default, the platform is looking for the web process type to start. You need
-to specify how to start the applicative server, and define it to display logs
+You need to specify how to start the applicative server, and define it to display logs
 on `stdout` to fit the [12-factor](https://12factor.net/) principles.
 
-### Using a WSGI Server
+This is acheived by create a file named `Procfile` at the root of the project with a web entry.
 
-Create a file named `Procfile` at the root of the project containing:
+Scalingo handle WSGI and ASGI server :
 
 ```yaml
 web: gunicorn myapp.wsgi --log-file -
 ```
 
-### Using a ASGI Server
-
-In order to make use of a ASGI server such as Uvicorn, create a file named
-`Procfile` at the root of the project containing:
-
 ```yaml
 web: gunicorn myapp.asgi --worker-class=uvicorn.workers.UvicornWorker --log-file -
 ```
 
-## Compile `.mo` Message Translation File
+{% note %}
+  For more details, check the documentation about [Procfile]({% post_url platform/app/2000-01-01-procfile %}).
+{% endnote %}
+
+### Start script
 
-If you are using `compilemessages` command to generate `.mo` file for gettext
-translations, that command needs to run before starting Gunicorn.
+A better practice is to use a start script, usually stored in bin directory. This
+will give you more flexibility on how to start your app.
```

Comment:
nitpick(non-blocking): I would say that the directory name should be formatted as code.


```suggestion
A better practice is to use a start script, usually stored in the `bin` directory. This
gives you more flexibility on how to start your app.
```